src/logger.js

```diff
@@ -8,7 +8,7 @@ var config = {
 	name: 'pact-node@' + pkg.version,
 	streams: [
 		{
-			level: 'info',
+			level: 'debug',
```

Comment:
Make this info again.  Maybe use an environment variable to flick between dev and release version?